Vagrantfile

```diff
@@ -124,7 +124,7 @@ cat << EOF > /home/vagrant/.ipython/kernels/toree-kernel/kernel.json
     "display_name": "Spark 1.5.1 (Scala 2.10.4)",
```

Comment:
We should change this `display_name` to `Toree`. Thoughts on whether we should add the version here?